src/main/java/uk/gov/pay/webhooks/deliveryqueue/dao/WebhookDeliveryQueueEntity.java

```diff
@@ -22,7 +22,7 @@
 
 @NamedQuery(
         name = WebhookDeliveryQueueEntity.NEXT_TO_SEND,
-        query = "select m from WebhookDeliveryQueueEntity m where :send_at > send_at and delivery_status = 'PENDING' order by send_at asc"
+        query = "select m from WebhookDeliveryQueueEntity m where :send_at > send_at and delivery_status = 'PENDING'"
```

Comment:
If there are a lot of failed webhooks, do we want to retry these last? or is it a different delivery status we have for failed attempts?

Reply:
Great question - the way it's currently architected if webhook message fail delivery attempt is marked terminally as 'FAILED' and a new delivery attempt is written for some time in the future and marked as 'PENDING' (in a db transaction for consistency).

So this query won't deal with anything in the future (ones which have failed and been rescheduled) it only deals with things that are scheduled for right now and were not picked up in the last few hundred ms since it last checked.